src/components/FilterPanel/FilterPanel.js

```diff
@@ -1,107 +1,60 @@
 /**
- * @file Filter Panel.
- * @copyright IBM Security 2019
+ * @file Filter panel component.
+ * @copyright IBM Security 2020
  */
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import FilterSearch from './FilterSearch';
-import FilterCategory from './FilterCategory';
-import {
-  filterDataPropTypes,
-  filterDataDefaultProps,
-  getFilterCategoriesArray,
-} from './FilterPanelUtilities';
+import classnames from 'classnames';
+
+import LegacyFilterPanel from './LEGACY_FilterPanel';
 import { getComponentNamespace } from '../../globals/namespace';
-import * as defaultLabels from '../../globals/nls';
+import deprecatedProp from '../../globals/prop-types';
 
 export const namespace = getComponentNamespace('filter-panel');
 
 const FilterPanel = props => {
-  const {
-    title,
-    filterData,
-    onChange,
-    filtersExpandLabel,
-    filtersCollapseLabel,
-    filterSearchLabel,
-    noFiltersResultsLabel,
-    labels,
-  } = props;
-  const componentLabels = {
-    ...defaultLabels.labels,
-    ...labels,
-    ...defaultLabels.filterFalsey({
-      FILTER_PANEL_CATEGORY_EXPAND_LABEL: filtersExpandLabel,
-      FILTER_PANEL_CATEGORY_COLLAPSE_LABEL: filtersCollapseLabel,
-      FILTER_PANEL_SEARCH_LABEL: filterSearchLabel,
-      FILTER_PANEL_SEARCH_NO_RESULTS_LABEL: noFiltersResultsLabel,
-    }),
-  };
+  const { title, children, className, filterData } = props;
+
+  if (filterData) {
+    return <LegacyFilterPanel {...props} />;
+  }
 
   return (
-    <aside className={namespace}>
+    <aside className={classnames(namespace, className)}>
```

Comment:
Instead of an `aside` could we use a `section`? The reason is because the DAP tool flags this for the following --

```

ASIDE elements or elements having a WAI-ARIA 'complementary' role must have a label specified with 'aria-label' or 'aria-labelledby'.?
--
ASIDE element or element having a WAI-ARIA 'complementary' role is missing a label specified with 'aria-label' or 'aria-labelledby'.

```

Related to this checkpoint: https://www.ibm.com/able/guidelines/ci162/bypass_blocks.html